Cargo.toml

```diff
@@ -11,7 +11,7 @@ members = [
     "examples/basic",
     "examples/basic_file",
     "examples/log_ssh",
-    "examples/streaming",
+    "examples/streaming", "tests/functionality/test_app",
```

Comment:
Change the path to `basic_test` (looks like you renamed the directory but didn't change this)